content/en/continuous_testing/environments/multiple_env.md

```diff
@@ -79,8 +79,7 @@ With this override, the URL `https://my-app.com/some/path` gets rewritten as `ht
 Notice that the URL path is not affected by the rewrite because it is not part of the substitution regex.
 
 <div class="alert alert-info">
-Apart from the pipe <code>|</code> syntax presented above, <code>startUrlSubstitutionRegex</code> also supports the sed syntax with modifiers: <code>s|&lt;regex&gt;|&lt;rewritting rule&gt;|&lt;modifiers&gt;</code>.</br></br>
-The sed syntax is often used with a slash <code>/</code> separator, for example: <code>s/&lt;regex&gt;/&lt;rewritting rule&gt;/&lt;modifier&gt;</code>. However, it can use any character as a delimiter. When working on a URL containing an abundant number of slashes, Datadog recommends using another character rather than escaping all slashes of the URL.
+Only the pipe <code>|</code> syntax is supported for URL substitution regexes. Sed syntax and other delimiter characters are not supported.
```

Comment:
We do support a sed-like syntax `s/<matcher>/<replacer>/`. What's not supported is the fact that `sed` supports any character as a delimiter. You just have to stick to it the 3 times, e.g. `s$<matcher>$<replacer>$`